src/scss/includes/panels/favorite_panel.scss

```diff
@@ -114,8 +114,11 @@
   padding-left: 14px;
 }
 
-.favorite_panel__item__image.icon:before{
+.favorite_panel__item__image.icon:before {
   line-height: 28px;
+  display: block;
+  height: 38px;
+  overflow: hidden;
```

Comment:
this is probably not supposed to be part of this PR :)

Reply:
yep, i failled something